src/core/services/slot-service.ts

```diff
@@ -254,7 +254,7 @@ class SlotService {
 	 */
 	private getGptAdSlot(id: string): AdSlot {
 		const prefix = context.get('custom.serverPrefix');
```

Comment:
Maybe put empty string as a fallback here? 

```suggestion
		const prefix = context.get('custom.serverPrefix') ?? '';
```

Reply:
No, because we want to skip the dot `.` too when server prefix is not set.